steps/bitbucket-server-post-build-status/0.0.8/step.yml

```diff
@@ -0,0 +1,85 @@
+title: Bitbucket server post build status
+summary: |
+  Post build status to bitbucket server
+description: |
+  Post build status to bitbucket server
+website: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status
+source_code_url: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status
+support_url: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/issues
+published_at: 2020-01-14T22:21:58.87928+01:00
+source:
+  git: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status.git
+  commit: 09749b47e4aa02c41e36069be447370497253799
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: true
+inputs:
+- domain: null
+  opts:
+    description: Full domain name without protocol eg 'my-domain.com'
+    is_required: true
+    summary: Full domain name without protocol eg 'my-domain.com'
+    title: Bitbucket Server domain name
+- opts:
+    description: The username used to make REST calls to bitbucket server
+    is_required: true
+    summary: The username used to make REST calls to bitbucket server
+    title: Bitbucket Server username
+  username: null
+- opts:
+    description: The password for the bitbucket server username
+    is_required: true
+    is_sensitive: true
+    summary: The password for the bitbucket server username
+    title: Bitbucket Server password
+  password: null
+- opts:
+    description: |-
+      If set, this step will set a specific status instead of reporting the current build status.
```

Comment:
`If set,` is not correct as input with value options, needs to have default value, please fix the description in the next step release.